backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxRuleApi.scala

```diff
@@ -52,6 +52,7 @@ private object VeloxRuleApi {
   def injectLegacy(injector: LegacyInjector): Unit = {
     // Gluten columnar: Transform rules.
     injector.injectTransform(_ => RemoveTransitions)
+    injector.injectTransform(_ => PushDownInputFileExpressionBeforeLeaf)
```

Comment:
Let's do a rename for the two rules. Since they seem to always be used together as a rule pair.

```suggestion
    injector.injectTransform(_ => PushDownInputFileExpression.PreOffload)
```